mongodb/src/main/scala/akka/stream/alpakka/mongodb/Mongo.scala

```diff
@@ -0,0 +1,16 @@
+package akka.stream.alpakka.mongodb
+
+import com.typesafe.config.{Config, ConfigFactory}
+import org.mongodb.scala.{MongoClient, MongoDatabase}
+
+object Mongo {
+  private val config: Config = ConfigFactory.load(getClass.getClassLoader, "application.conf")
+
+  private val dbName: String = config.getString("mongo.name")
+  private val dbAddress: String = config.getString("mongo.host")
+  private val dbPort: String = config.getString("mongo.port")
+
+  val client: MongoClient = MongoClient(s"mongodb://$dbAddress:$dbPort")
```

Comment:
If the client is to be cached it should be bound to the stage lifecycle, not put in a global singleton. This will leak the client between tests, and keep it in memory even if there is no more use of it.